### PR TITLE
adding SSD

### PR DESCRIPTION
**Pull request by nv-kkudrynski**

### CODE REVIEW

**Comment by nv-kkudrynski** [comment]
To be safe with copyrights, I added visualization result of our own image in featured_image_2 placeholder. It shows the idea. Is that ok?

File: nvidia_deeplearningexamples_ssd.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+```python
+uris = [
+    'http://images.cocodataset.org/val2017/000000397133.jpg',
+    'http://images.cocodataset.org/val2017/000000037777.jpg',
+    'http://images.cocodataset.org/val2017/000000252219.jpg'
+]
+```
+Format the images to comply with the network input and convert them to tensor.
+```python
+inputs = [utils.prepare_input(uri) for uri in uris]
```
Comment (soumith) [inline nvidia_deeplearningexamples_ssd.md]: can you clarify or specify (maybe just add a comment), how does one specify a local image (i.e. not a `uri`)

File: nvidia_deeplearningexamples_ssd.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+tensor = utils.prepare_tensor(inputs, precision == 'fp16')
+```
+
+
+Run the SSD network to perform object detection.
+```python
+with torch.no_grad():
+    detections_batch = ssd_model.forward(tensor)
```
Comment (soumith) [inline nvidia_deeplearningexamples_ssd.md]: isn't it `ssd_model(tensor)`

**Comment by soumith** [comment]
yes that's fine. just asked for some minor clarifications in-line, but otherwise this is good to merge

File: nvidia_deeplearningexamples_ssd.md (hunk 1 above)
Comment (nv-kkudrynski) [reply nvidia_deeplearningexamples_ssd.md]: yes. you can put a local path here straightforward. added a comment about it.

File: nvidia_deeplearningexamples_ssd.md (hunk 2 above)
Comment (nv-kkudrynski) [reply nvidia_deeplearningexamples_ssd.md]: corrected

**Comment by soumith** [comment]
thanks @nv-kkudrynski !

**Pull request merged**